recipe/meta.yaml

```diff
@@ -1,48 +1,62 @@
 {% set name = "starlette" %}
-{% set version = "0.17.0" %}
+{% set version = "0.27.0" %}
 
 package:
   name: {{ name|lower }}-recipe
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 31a889e7d7bf487f70d9d197ed7efadb47fa938c58626ed93e381480833c5b84
+  sha256: 6a6b0d042acb8d469a01eba54e9cda6cbd24ac602c4cd016723117d6a7e73b75
 
 build:
   number: 0
+  skip: true  # [py<37]
+
+requirements:
+  host:
+    - python
+    - pip
+    - hatchling
 
 outputs:
   - name: {{ name }}
     build:
-      noarch: python
-    script: build_base.sh
+      script: python -m pip install . --no-deps --no-build-isolation -vv
     requirements:
       host:
-        - python >=3.6
+        - python
         - pip
+        - hatchling
       run:
-        - python >=3.6
-        - anyio >=3.0.0,<4.0.0a
+        - python
+        - anyio >=3.4.0,<5
+        - typing_extensions >=3.10.0  # [py<310]
     test:
+      imports:
+        - starlette
+      requires:
+        - pip
       commands:
-        - python -c "from starlette import *"
+        - pip check
 
   - name: {{ name }}-full
     build:
-      noarch: python
+      script: echo Nothing to do.
```

Comment:
I do not checked prefect logs, so I may be wrong here, but I believe that you need a `# [s390x]` here as well, otherwise it would hide the test script from every platform. could you check please?

Reply:
Yes, I skip s390x. As I'm building for the stage channel, you can check outputs and artifacts here https://anaconda.org/sk_test/repo